.github/workflows/publish-to-test-pypi.yml

```diff
@@ -13,16 +13,12 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: "3.x"
-    - name: Install pypa/build
-      run: >-
-        python3 -m
-        pip install
-        build
-        --user
-    - name: Run Unit Tests
-      run: |
-        pip install pytest pytest-cov requests-mock requests_toolbelt pandas
-        pytest tests/ --doctest-modules --cov --cov-report=xml:out/coverage.xml
+    - name: Install Poetry
+      uses: abatilo/actions-poetry@v2
+    - name: Install Dependencies
```

Comment:
Can the dependencies get cached? Seems like you can hash on the poetry.lock

Reply:
Yes, you can. I'm currently working on some CI stuff, like getting the test PyPI deployment working. I intend to add caching at that point.